lib/watcher/appmetrics.js

```diff
@@ -0,0 +1,76 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+var UNUSED_PROBES = [
+  'eventloop',
+  'profiling',
+  'http',
+  'mongo',
+  'socketio',
+  'mqlight',
+  'postgresql',
+  'mqtt',
+  'mysql',
+  'redis',
+  'memcached',
+  'oracledb',
+  'strong-oracle',
+  'requests',
+  'trace',
+];
+
+exports.worker = function(handle) {
+  var appmetrics = require('appmetrics');
```

Comment:
you have to find a way to use `../adapter`, so that `sl-run --no-profile` causes no appmetrics to be required